From 29f5cf838e70ac8f8dede0ff61b643f53b0f8fa5 Mon Sep 17 00:00:00 2001
From: Fernando Fernandez <fernando@softwareperonista.com.ar>
Date: Wed, 21 Sep 2016 22:33:26 -0300
Subject: [PATCH] fix freerdp check

---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0dfc0db..35e5f7b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -58,7 +58,7 @@ AS_IF([test "x$have_ssh" = "xyes"],
 AM_CONDITIONAL([VINAGRE_ENABLE_SSH], [test "x$have_ssh" = "xyes"])
 
 # Whether to enable support for RDP.
-RDP_DEPS="freerdp x11"
+RDP_DEPS="freerdp2 x11"
 AC_ARG_ENABLE([rdp],
   [AS_HELP_STRING([--disable-rdp],
     [Disable Remote Desktop Protocol (RDP) support])])
@@ -66,7 +66,7 @@ AC_ARG_ENABLE([rdp],
 AS_IF([test "x$enable_rdp" != "xno"],
   [PKG_CHECK_EXISTS([$RDP_DEPS],
     [have_rdp=yes
-     PKG_CHECK_EXISTS(freerdp >= 1.1,
+     PKG_CHECK_EXISTS(freerdp2 >= 1.1,
        [AC_DEFINE([HAVE_FREERDP_1_1], [1], [FreeRDP is of version 1.1 or newer])], [])],
     [have_rdp=no])],
   [have_rdp=no])
-- 
2.10.0

