Description: Fix --enable-exact-rooting confflag
Author: Rico Tzschichholz <ricotz@ubuntu.com>
Forwarded: no
Last-Update: 2014-12-28

---

--- a/js/src/configure.in	Sat Dec 27 03:14:56 2014 -0800
+++ b/js/src/configure.in	Sun Dec 28 21:58:46 2014 +0100
@@ -3257,7 +3262,7 @@
 dnl ========================================================
 dnl = Use exact stack rooting for GC
 dnl ========================================================
-MOZ_ARG_DISABLE_BOOL(exact-rooting,
+MOZ_ARG_ENABLE_BOOL(exact-rooting,
 [  --enable-exact-rooting   Enable use of conservative stack scanning for GC],
     JSGC_USE_EXACT_ROOTING=1 ,
     JSGC_USE_EXACT_ROOTING= )

